app/models/normalize_primo_articles.rb

```diff
@@ -0,0 +1,23 @@
+# Tranforms results from {SearchPrimo} into normalized {Result}s
+# for metadata that is relevant for article type results
+class NormalizePrimoArticles
+  def initialize(record)
+    @record = record
+  end
+
+  def article_metadata(result)
+    result.citation = numbering
+    result.in = journal_title
+    result
+  end
+
+  def journal_title
+    return unless @record['pnx']['addata']['jtitle']
+    @record['pnx']['addata']['jtitle'].join('')
+  end
+
+  def numbering
+    return unless @record['pnx']['addata']['volume'] && @record['pnx']['addata']['issue']
```

Comment:
I don't have an example record, but I assume some journals have volumes but not issues.

I'm not opposed to leaving this as is with an additional ticket to return to it though.

Reply:
Oh, that's a good point. I'll refactor to account for that.